src/loadgenerator/locust_tasks/sre_recipe_load_tasks.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from locust import task, between, HttpUser
+
+
+PRODUCTS = [
+    '0PUK6V6EV0',
+    '1YMWWN1N4O',
+    '2ZYFJ3GM2N',
+    '66VCHSJNUP',
+    '6E92ZMYYFZ',
+    '9SIQT8TOJO',
+    'L9ECAV7KIM',
+    'LS4PSXUNUM',
+    'OLJCESPC7Z',
```

Comment:
2 things:

1) maybe we can centralize the list of products in one place, accessible both by the sample app microservices as well as loadgen.

2) wherever this list is created, we should add inline comment saying which product of hipstershop it represents so it's easier to track the code with context.

Reply:
1. ACKed. It seems that doing that will involve more changes affecting multiple directories to ensure the data files are correctly packaged across all the builds and images, and resolve any cross folder python import issues. I will leave it out for this PR. 
2. Done.